docs/application/tizen-studio/release-notes/5-0-release-notes.md

```diff
@@ -0,0 +1,74 @@
+# Tizen Studio 5.0 Release Notes
+
+- Release Date: Oct 31, 2022
+
+## IDE and tools
+
+### New feature
+
+- WHPX support on Windows
+  - WHPX support has been added in Tizen emulators. Users can now use either HAXM or WHPX accelerators to launch Tizen emulators on windows.
+- macOS notarization support for Tizen installer
+  - Tizen installers are now officialy notarized with Apple. Users can install tizen studio on macOS without any notarization errors.
+- Ubuntu 22.04 support
+  - Tizen Studio is now supported on ubuntu 22.04.
+  - Visual Studio Code Extension for Tizen is now supported on ubuntu 22.04.
+- CLI
+  - support for non samsung vendor TV devices.
+  - Added support for RPK projects.
+- VSCode
+  - Added support for RPK projects.
+- Visual Studio (Windows)
+  - Added support to install Tizen dotnet workload in non-admin mode.
+  - Intellisense support has been added for Tizen web project config.xml file and workspace yaml file.
+
+### Fixed bugs
+  - Added the option in CLI to include the files outside app directory while packaging.
+  - Fixed the error of on-demand packages not getting installed in Visual Studio when multiple target devices are launched.
+  - Fixed the issue of Multi app project not getting packaged in CLI, If static lib is the main project. 
+  - Fixed the issue of directories not being created in CLI packaging command when -o option is passed.
+  - Valgrind profiling support has been removed from tizen studio as its not supported in tizen platform anymore.
+  - IOT setup manager support has been removed from tizen studio.
+
+### Known issues
+
+  - In macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
+  - In case you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+  - In macOS from Catalina and above versions, Native Templates-5.5 will not build with CLI when the compiler is set to **gcc**.
+- Common
+  - If you install Tizen Studio in a directory that requires administrator privileges for access, an alert will be given. . For example, for **C:\ProgramFiles** location administrator privilege is needed to run Tizen SDK tools. Tizen Installer and Baseline SDK Installer alerts you if you try to install it into such a directory.
+- Web and Native IDE
+  - You can create unit tests for Tizen 2.3.2 and higher version projects only. Unit testing for older versions is not supported.
+- Web IDE
+  - The preview tab in the Web Page Editor sometimes does not appear properly. Use an alternative feature, named Web SDK HTML Editor, which has enhanced features compared to the Web Page Editor. Instead of the preview tab in the Web Page Editor, use the preview feature **Ctrl + 4** of the Web SDK HTML Editor.

```

Comment:
```suggestion
  - The preview tab in the Web Page Editor sometimes does not appear properly. In this case, you can use an alternative feature, named Web SDK HTML Editor, which has enhanced features compared to the Web Page Editor. Instead of the preview tab in the Web Page Editor, use the preview feature **Ctrl + 4** of the Web SDK HTML Editor.

```

Reply:
Fixed as per comments